.github/workflows/e2e-fixture-test.yml

```
@@ -0,0 +1,31 @@
+name: E2E Fixture Test
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

We should pin this, similar to how we have done in other workflows: https://github.com/metal3-io/baremetal-operator/blob/74bfd27aeb2490263013bc39ca10a39223b9a11f/.github/workflows/dependabot.yml#L21

```suggestion
    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```